resources/migrations/0120_merge_20231117_1306.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 2.2.11 on 2023-11-17 11:06
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
You could also delete the migrations 0117 and 0118 added by this PR and recreate them (after reverting them locally of course) while on top of the `tre-varaamo-qa` branch. Then merge migrations wouldn't be needed.